docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Upgrade a Quickstart Network
+id: upgrade-quickstart-network
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document explains the process for backing up, and upgrading your Ziti Controller and Ziti Router(s).
+
+<Tabs groupId="backup-network">
+<TabItem value="non-docker-quickstart" label="Non-Docker Quickstart">
+
+## Backup Existing Network
+
+While you can certainly back up every file relating to your network, there are only a few that would be necessary. To
+back up the necessary files follow these steps:
+
+1. Backup the database:
+    1. Run the following
+   ```
+   ziti edge db snapshot
+   ```
+    1. This will create a backup file in the same directory as your controller's DB file. It will have the same name as
+       the controller DB file (usually `ctrl.db`) with a date and timestamp appended to it in the following format
+       `ctrl.db-YYYYMMDD-HHMMSS`. If you're not sure where your DB file is located, you can find the location in
+       the `db`
+       section of your controller config.
+1. Backup the controller PKI.
+    1. Copy the PKI directory to a safe backup location (Default path: `$ZITI_HOME/quickstart/$(hostname -s)/pki/`).
+1. Backup the controller config file (default path: `$ZITI_HOME/quickstart/$(hostname -s)/$(hostname -s).yaml`).
+1. Backup the environment file (default path `$ZITI_HOME/quickstart/$(hostname -s)/$(hostname -s).env`).
+1. Backup any router config files.
+    1. If you used one of our quickstarts, the single edge router created defaults
+       to `$ZITI_HOME/quickstart/$(hostname -s)/$(hostname -s)-edge-router.yaml`
+
+## Stop Existing Services
+
+If your network is using services, stop those services.
+
+<Tabs groupId="stop-services">
+<TabItem value="linux-services" label="systemd (Linux)">
+
+```
+sudo systemctl stop ziti-router ziti-controller
+```
+
+</TabItem>
+<TabItem value="mac-services" label="launchd (Mac)">
+
+```
+launchctl stop ziti-router ziti-controller
+```
+
+</TabItem>
+</Tabs>
+
+## Obtain the Desired Binary
+
+Ziti releases can be found [here](https://github.com/openziti/ziti/releases). There is a helpful Ziti CLI script
+function called `getZiti` that will obtain the binary and will determine the correct architecture and OS for you.
+
+Here is the process for using `getZiti` to obtain the latest binary. You may optionally add `yes` as an argument to
+the `getZiti` function to have the new ziti binary automatically added to your $PATH
+
+```
+source /dev/stdin <<< "$(wget -qO- https://get.openziti.io/quick/ziti-cli-functions.sh)"
```

Comment:
I kinda feel like this code snippet should be a reusable component. It appears twice on this page and also in our quickstarts... feels like that's enough to make a shared component to me. If it's possible, one component for all shared snippets like this might be good?